CHANGELOG.md

```diff
@@ -2,39 +2,40 @@
 
 ## 4.0.0 - UNRELEASED
 
-- BREAKING: Map `multi-merge` utility and all `add-*` functions
-  for adding one or more token maps (eg `add-sizes`, `add-colors`, etc)
+- BREAKING: Map `multi-merge()` utility and all `add-*` functions
+  for adding one or more token maps (e.g. `add-sizes()`, `add-colors()`, etc)
   now perform a deep merge of the token maps.
   Previously a shallow merge was performed,
   [which could result in removing nested values from maps](https://github.com/oddbird/accoutrement/issues/50).
 
 - [Tokens][token]:
 
-  - NEW: `map-compile` and `map-compile-with` functions for compiling
+  - NEW: `map-compile()` and `map-compile-with()` functions for compiling
     tokens in Accoutrement maps and formatting them for static exports.
 
 - [Animate][animate]:
 
   - NEW: `compile-*` functions provide an export option for respective token maps
-    (see: `compile-changes`, `compile-easing`, `compile-times`).
+    (see: `compile-changes()`, `compile-easing()`, `compile-times()`).
```

Comment:
Maybe we can add links to these since we say "see this thing"? It will make it a little easier for someone to find them. I realize we aren't linking to all new or revised functions, but we probably could?

Reply:
@stacyk Ah, yes. Adding links definitely make sense here and I think it makes to the changlelog more user friendly in general to add them to other references someone might want to jump to. I added in these links for now and will add more for the recent changes.

Reply:
Looks good and was easier to use 🎉